### PR TITLE
#20 - Create-HomeScreen

### LINKED ISSUE #20: Create Home Screen
image

### PR DESCRIPTION
**Pull request by Koushith**
## Description
- Adds Home Screen.
- Adds TopBar Component.
- Fixed overflow issues.
- Fixed headers on smaller devices.

## Screenshot
https://www.loom.com/share/278cf2c2b8104c54b6522ff4e1427474

image
image

## Platform tested
- [x] web.
- [x] Mobile.

### CODE REVIEW

File: src/screens/home/home.screen.tsx [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
-  margin: 10px;
-`;
 export const HomeScreen: React.FC = () => {
   return (
-    <div>
-      <NavBar />
-      {/* <BottomTab /> */}
-    </div>
+    <HomeScreenContainer>
+      <Title variant='contentHeader'>Safes</Title>
```
Comment (dsouzaedison) [inline src/screens/home/home.screen.tsx]: Is this hard-coded?

File: src/screens/home/home.screen.tsx [hunk 2]
```diff
@@ -1,18 +1,17 @@
...
+      <CardsContainer>
+        <SafeCard hCenter vCenter>
+          <IconSvg name='add' size='xLarge' />
+          <SafeText variant='content'>Create Safe</SafeText>
```
Comment (dsouzaedison) [inline src/screens/home/home.screen.tsx]: Localise strings

**Review by dsouzaedison — CHANGES_REQUESTED** [review]

File: src/screens/home/home.screen.tsx (hunk 2 above)
Comment (Koushith) [reply src/screens/home/home.screen.tsx]: Added in Create Card Component's PR. - https://github.com/safient/safient-client/pull/59

File: src/screens/home/home.screen.tsx (hunk 1 above)
Comment (Koushith) [reply src/screens/home/home.screen.tsx]: Removed all Hardcoded Values in the latest commit.

**Review by koshikraj — APPROVED** [review]

**Pull request merged**